src/main/java/hw01/Human.java

```diff
@@ -123,15 +124,29 @@ public void describePet() {
 
     }
 
+    public void feedPet() {
+        if (pet.isHungury() == false) {
```

Comment:
delete this isHungury field and refactor the code, pls read documentation carefully, don't add additional fields, use fields which is declared on documentation, don't do overengineering